client/customer-effort-score-tracks/data/actions.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import TYPES from './action-types';
+
+/**
+ * Initialize the state
+ *
+ * @param {Object} queue	initial queue
+ */
+export function setCesSurveyTracksQueue( queue ) {
+	return {
+		type: TYPES.SET_CES_SURVEY_TRACKS_QUEUE,
+		queue,
+	};
+}
+
+/**
+ * Add a new CES track to the state.
+ *
+ * @param {string} action action name for the survey
+ * @param {string} label label for the snackback
+ * @param {string} pageNow value of window.pagenow
+ * @param {string} adminPage value of window.adminpage
+ * @param {string} onsubmit_label label for the snackback onsubmit
+ */
+export function addCesSurveyTrack(
+	action,
+	label,
+	pageNow = window.pagenow,
+	adminPage = window.adminpage,
+	onsubmit_label = undefined
+) {
+	return {
+		type: TYPES.ADD_CES_SURVEY_TRACK,
+		action,
+		label,
+		pageNow,
+		adminPage,
+		onsubmit_label,
+	};
+}
+
+/**
+ * Add a new CES survey track for the pages in Analytics menu
+ */
+export function addCesSurveyTrackForAnalytics() {
```

Comment:
Can you change this to call `addCesSurveyTrack`? I think that would make it clearer that `addCesSurveyTrack` is the one that should be used by default.

Reply:
Good catch! 👍 changed it in https://github.com/woocommerce/woocommerce-admin/pull/5649/commits/a1a9552ced71db7a1ad5708892dc30e8bd789653